qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-PhoneNumberValidation/phoneNumberValidation.feature

```diff
@@ -0,0 +1,24 @@
+Feature:Phone Number validation
+
+  Background:
+    Given user goes to find patient record app
+   
+  @selenium
+  @phoneNumberValidation
+  Scenario: Phone Number validation
+    When user searches for patient using patient identifier
+    And user clicks on the first patient
```

Comment:
Does the search using patient identifier return more than one patient?

Reply:
It returns many patients but we only need the first patient.

Reply:
Can you give an example of the patient identifier that i can use on `https://qa-refapp.openmrs.org/` to get many patients returned?

Reply:
@dkayiwa here we dont care about how many gets returned, we only care about the availability of patients in the db and click on the first patient returned.

Reply:
There is no specific patient identifier that returns many patients unless its mapped which is not something we would love to look into.

Reply:
Click on the first one means it could potentially return more than one.

Reply:
Click on the first patient returns only one patientdashboard unless this patient is a merged one which is a rare case.

Reply:
When you say click on first, you essentially mean that more than one is returned but choose the first among those that are returned.

Reply:
Thats true @dkayiwa . Does it have any problem ?.

Reply:
Can you give me a patient identifier which returns more than one patient on this server? https://qa-refapp.openmrs.org/

Reply:
We dont have that patient identifier.

Reply:
Then what do you base on to say that a patient identifier can return more than one patient?

Reply:
Apologies.

Reply:
i think this means we should change every feature file with that naming since each patient identifier returns one patient.